05_classify_mutations_multimodal/nbconverted/plot_multimodal_results.py

```diff
@@ -27,19 +27,26 @@
 # In[2]:
 
 
-# set results directory
-results_dir = Path(
-    # cfg.results_dirs['multimodal'],
-    # 'pilot_results',
-    '../04_classify_mutations_multimodal',
-    'results',
-    'pilot_results_all_feats',
-    'gene'
-).resolve()
-
 # if True, save figures to ./images directory
 SAVE_FIGS = True
 
+# if True, use raw feature results, else use compressed features
+RAW_FEATS = True
```

Comment:
Is the idea that there are two datasets (or feature spaces I  guess) and users will have to run the notebook twice with `RAW_FEATS` set to True and then False to see both sets of figures? If so, why not just do two notebooks, or a function with a flag that you call twice so that all the figures are generated without manual intervention?

Reply:
The figures generated with `RAW_FEATS=False` are in the main paper, and the figures generated with `RAW_FEATS=True` go in the supplement (which I guess should probably be in a comment somewhere). Other than the results data the notebook pulls from, there's no difference in how the figures are generated.

I guess I could make this into two notebooks, but that would result in a lot of duplicate code that I'd have to update in two places anytime I change things. Maybe I can find a way to call a function twice with a flag and generate all the figures at once - let me think about it.